Code/training_common_utils.py

```diff
@@ -44,7 +55,7 @@ def build_data_loaders(base_label_transform, dlparams):
     data_training = ChexpertDataset(
         csv_file='../Data/CheXpert-v1.0-small/train.csv',
         root_dir='../Data',
-        image_transform=image_preprocessing,
+        image_transform=image_augmented_preprocessing,
```

Comment:
Wanna change line 51 as well?

Reply:
No, but for a good reason. The data_training dataset is being initialized with image_augmented_preprocessing because the augmented preprocessing includes the `RandomAffineTransform` thing. The 'validation' set should be the original data, and shouldn't have anything randomized applied to it... instead sticking to the raw images (after being resized/normalized/histogram equalized, and and gaussian blurred. 

I need to read up more on the gaussian blur, but to my understanding these are all non-random transforms, and can thus be applied to all images before the model is ran. 

the randomAffine is a randomly applied transform causing (i think) rotation/translation/etc... along a couple different random distributions (i probably should double check the docs on that one too).